### PR TITLE
PEAR-2162 - Add NCI OCPL Government Shutdown Banner to GDC Documentation Site

### PR DESCRIPTION
**Pull request by amlehman**
Adds the script specified in the ticket. Wasn't able to get the banner actually to show up in my development environment so not sure if more steps are needed?

### CODE REVIEW

File: overrides/home.html [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {% extends "base.html" %}
 
+{% block extrahead %}
+<script src="https://assets.adobedtm.com/6a4249cd0a2c/785de09de161/launch-70d67a6a40a8.min.js" async></script
+{% endblock %}
```
Comment (philtom-ctds) [inline overrides/home.html:5]: Oh... this is... :laugh-melt:

We already use adobe tag manager for analytics. I'm genuinely curious if having two launch script is going to work as expected.

The analytics scripts were added in this commit:
https://github.com/NCI-GDC/gdc-docs/commit/06f9cec83d19d8203cdf40c83a021758f145bcac

The other script is added to `overrides/partials/integrations/analytics/custom.html`, which is the partial for adding analytic scripts to the header.  We could probably put both scripts in there.  Maybe put comments in there to let us know which is which.

In that pr, there's also a `docs/js/analytics-after-load.js` script that is loaded at the bottom of each body.  That script triggers the tag manager (`_satellite`).  I'm wondering is both of the adobedtm scripts are going to stop on each others feet.

Comment (philtom-ctds) [reply overrides/home.html:5]: i just took a look at the two launch scripts. the second one will clobber the first. they are both IIFE-style modules that start with
```
 (window._satellite = window._satellite || {}),
   (window._satellite.container = {
```
The object being assigned to `container` is different in the two scripts.  They look like all the config data for which domains it should run on, which adobe environment ids it should attach to, and other mysteries.  Also, if I'm interpreting these things correctly, the gov't shutdown banner won't trigger on the gdc domains.

Comment (philtom-ctds) [reply overrides/home.html:5]: I just read the attachment in the ticket.  It has instructions for sites that already use Adobe Launch.  The OCPL Analytics team needs to update things on their side.  Then, when that's done, we replace the previous launch script with the new launch script. That means that this launch script should replace the one in `overrides/partials/integrations/analytics/custom.html`.

This pr should not be merged until we've confirmed that the OCPL Analytics teams has done their part first.

**Comment by amlehman** [comment]
Here she is:
Screenshot 2024-10-02 at 12 03 02 PM

**Review by philtom-ctds — APPROVED** [review]

**Pull request merged**